### PR TITLE
Migrate to using twitter search instead of twitter stream

### PR DESCRIPTION
**Pull request by esirK**
## Description

- Clean up codebase. 
- Use Twitter API search instead of Twitter Stream
- 

Fixes # (issue)

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

**Comment by kilemensi** [comment]
> Use Twitter API search instead of Twitter Stream
>
Are we replacing _stream_ with _search_ or complementing it @esirK ?

**Comment by esirK** [comment]
> Are we replacing _stream_ with _search_ or complementing it @esirK ?

Right now we are just using _search_ . The last time we were using _stream_ it seemed to clog up other celery tasks so I temporarily disabled it. The code is still intact just the streaming task itself has been removed.  @kilemensi

File: debunkbot/celeryapp.py [hunk 1]
```diff
@@ -54,14 +54,14 @@
...
-        "task": "stream_listener",
-        "schedule": crontab(minute=f"*/{DEBUNKBOT_RESTART_STREAM_LISTENER_INTERVAL}"),
-    },
     "check_tweet_metrics": {
         "task": "check_tweet_metrics",
         "schedule": crontab(minute=f"*/{DEBUNKBOT_CHECK_TWEETS_METRICS_INTERVAL}"),
     },
+    "claims_search": {
+        "task": "claims_search",
+        "schedule": crontab(minute=f"*/{20}"),
```
Comment (kilemensi) [inline debunkbot/celeryapp.py:63]: Why hard-coding `20`?

File: debunkbot/management/commands/send_replies.py [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
+from debunkbot.twitter.process_tweet import process_tweet
 
 
 class Command(BaseCommand):
     help = "Management command that sends replies to tweets with debunked urls"
 
     def handle(self, *args, **options):
         self.stdout.write(self.style.SUCCESS("Sending reply to selected tweet."))
-        process_stream()
+        process_tweet()
```
Comment (kilemensi) [inline debunkbot/management/commands/send_replies.py:11]: Which tweet are we replying/processing?

File: debunkbot/tasks.py [hunk 3]
```diff
@@ -37,8 +42,23 @@ def stream_listener():
         logger.info("No claims in the database.")
 
 
+@app.task(name="claims_search", task_ignore_result=True)
+def claims_search():
```
Comment (kilemensi) [inline debunkbot/tasks.py:46]: search_claims

File: debunkbot/tasks.py [hunk 4]
```diff
@@ -18,6 +21,8 @@
 
 logger = get_task_logger(__name__)
 
+api = create_connection()
+
 
 @app.task(name="stream_listener", task_ignore_result=True)
 def stream_listener():
```
Comment (kilemensi) [inline debunkbot/tasks.py:28]: We no longer have stream right?

**Review by kilemensi — COMMENTED** [review]
👍🏾

File: debunkbot/tasks.py (hunk 4 above)
Comment (esirK) [reply debunkbot/tasks.py:28]: Yeah. I'll remove it.

File: debunkbot/celeryapp.py (hunk 1 above)
Comment (esirK) [reply debunkbot/celeryapp.py:63]: Will fix it.

File: .env.sample [hunk 5]
```diff
@@ -21,7 +21,7 @@ DEBUNKBOT_CHECK_TWEETS_METRICS_INTERVAL=
 DEBUNKBOT_CHECK_IMPACT_INTERVAL=
 DEBUNKBOT_BOT_PULL_CLAIMS_INTERVAL=
 DEBUNKBOT_BOT_UPDATE_GSHEET_INTERVAL=
-DEBUNKBOT_RESTART_STREAM_LISTENER_INTERVAL=
```
Comment (kilemensi) [inline .env.sample:24]: Some variables are named `DEBUNKBOT_BOT_XXX`. Any reason for the `_BOT_`?

**Review by kilemensi — APPROVED** [review]
🚀 

----

Do a search for "stream" and update accordingly (in README and test_claims_handler).

**Pull request merged**